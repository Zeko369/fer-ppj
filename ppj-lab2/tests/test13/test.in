IDN 1 a
OP_PRIDRUZI 1 =
BROJ 1 0
KR_ZA 2 za
IDN 2 i
KR_OD 2 od
OP_MINUS 2 -
BROJ 2 3
KR_DO 2 do
BROJ 2 5
IDN 3 a
OP_PRIDRUZI 3 =
IDN 3 a
OP_PLUS 3 +
IDN 3 i
KR_AZ 4 az
