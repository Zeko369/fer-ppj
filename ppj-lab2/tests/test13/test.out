<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 a
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 0
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <za_petlja>
     KR_ZA 2 za
     IDN 2 i
     KR_OD 2 od
     <E>
      <T>
       <P>
        OP_MINUS 2 -
        <P>
         BROJ 2 3
       <T_lista>
        $
      <E_lista>
       $
     KR_DO 2 do
     <E>
      <T>
       <P>
        BROJ 2 5
       <T_lista>
        $
      <E_lista>
       $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 3 a
        OP_PRIDRUZI 3 =
        <E>
         <T>
          <P>
           IDN 3 a
          <T_lista>
           $
         <E_lista>
          OP_PLUS 3 +
          <E>
           <T>
            <P>
             IDN 3 i
            <T_lista>
             $
           <E_lista>
            $
      <lista_naredbi>
       $
     KR_AZ 4 az
   <lista_naredbi>
    $
